hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -856,11 +856,23 @@ void unassign(byte[] regionName, boolean force)
    * the request was submitted successfully. We need to check logs for the details of which regions
    * were split/merged.
    *
-   * @return <code>true</code> if region normalizer ran, <code>false</code> otherwise.
+   * @return {@code true} if region normalizer ran, {@code false} otherwise.
    * @throws IOException if a remote or network exception occurs
    */
   boolean normalize() throws IOException;
```

Comment:
nit: could we add a default method here that calls `normalize(NormalizeTableFilterParams)` with a default one? seems like we could clean up some stuff in the various *Admin implementations and `RequestConverter`

Reply:
Oh, good point. I basically never think of interfaces as supporting `default` implementations. That's a decent plan, I believe.